src/main/webapp/partials/instructorHelpFAQ.jsp

```diff
@@ -1,198 +1,204 @@
             <br>
-            <li>
-                <b>
-                    <a name="faq11" id="question">How are the contribution numbers calculated?
-                    </a>
-                </b>
-                <div class="helpSectionContent">
-                    You can find the details of the contribution calculation scheme
-                    <a href="https://docs.google.com/document/d/1hjQQHYM3YId0EUSrGnJWG5AeFpDD_G7xg_d--7jg3vU/pub?embedded=true#h.n5u2xs6z9y0g">here</a>.
-                </div>
-            </li>
             <br>
-            <li>
-                <b>
-                    <a name="faq12" id="question">Is it compulsory for students to use Google accounts?</a>
-                </b>
-                <div class="helpSectionContent">
-                    Student can submit feedback and view results without Google accounts. TEAMMATES send unique URL for each of those. But it is more convenient for them if they use Google accounts.
-                    <br>
-                    <br>
-                </div>
-            </li>
-            <li>
-                <b>
-                    <a name="faq13" id="question">What if my course doesn’t have teams?</a>
-                </b>
-                <div class="helpSectionContent">
-                    They can use a dummy value for the ‘Team’ column.
-                    <br>
-                    <br>
-                </div>
-            </li>
-            <li>
-                <b>
-                    <a name="faq14" id="question">Is there a size limit for a course?</a>
-                </b>
-                <div class="helpSectionContent">
-                    No. But courses bigger than 100 students need to divide the course into sections so that TEAMMATES know how to paginate reports when the entire report is too big to show in one go.
-                    <br>
-                    <br>
-                </div>
-            </li>
-        </ul>
-        <p align="right">
-            <a href="#Top">Back to Top
-            </a>
-        </p>
-    </div>
-    <div class="separate-content-holder">
-        <hr>
-    </div>
+          </div>
+          In the calculation, we do not allow (a) to affect (b). We use (b) to calculate the average perceived contribution for each student. A more detailed version of this calculation can be found
+          <a href="https://docs.google.com/document/d/1hjQQHYM3YId0EUSrGnJWG5AeFpDD_G7xg_d--7jg3vU/pub?embedded=true#h.n5u2xs6z9y0g">here</a>.
+        </div>
+      </li>
+      <br>
+      <li>
+        <b>
+          <a name="faq11" id="question">
+            How are the contribution numbers calculated?
+          </a>
+        </b>
+        <div class="helpSectionContent">
+          You can find the details of the contribution calculation scheme
+          <a href="https://docs.google.com/document/d/1hjQQHYM3YId0EUSrGnJWG5AeFpDD_G7xg_d--7jg3vU/pub?embedded=true#h.n5u2xs6z9y0g">here</a>.
+        </div>
+      </li>
+        <br>
```

Comment:
Missed this: remove 2 spaces from this line.

Reply:
Done